src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/MySqlDataSourceItemFixture.cs

```diff
@@ -0,0 +1,40 @@
+using Reveal.Sdk.Dom.Core.Extensions;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class MySqlDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_WhenConstructed()
+        {
+            // Arrange
+            var title = "Test Item";
+            var dataSource = new MySQLDataSource();
+
+            // Act
+            var item = new MySqlDataSourceItem(title, dataSource);
+
+            // Assert
+            Assert.Equal(title, item.Title);
+            Assert.Equal(dataSource, item.DataSource);
```

Comment:
Should use Theory to increase the coverage
Need more Asserts:
- DataSource.Title = Title
- DataSourceId = dataSource.Id